shuttle/templates/schedules/request_shuttle.html

```diff
@@ -9,33 +9,61 @@ <h4>Select a location and time below to request the shuttle:</h4>
     </div>
 
     <div class="center-page">
-        <div class="align-side-by-side">
-            <div class="dropdown">
-                <button class="btn btn-primary btn-lg dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
-                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-                    <a class="dropdown-item" href="#">*Insert location here*</a>
-                    <a class="dropdown-item" href="#">*Insert more locations*</a>
-                </div>
+        <div class="dropdown">
+            <button class="btn btn-primary btn-lg dropdown-toggle location-menu" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Location</button>
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <a class="dropdown-item" href="#">CLC</a>
+                <a class="dropdown-item" href="#">Anderson</a>
+                <a class="dropdown-item" href="#">Pine Tree</a>
+                <a class="dropdown-item" href="#">Kresge</a>
+                <a class="dropdown-item" href="#">Scandia</a>
+                <a class="dropdown-item" href="#">North</a>
+                <a class="dropdown-item" href="#">Rosedale Center</a>
+                <a class="dropdown-item" href="#">HealthPartner Clinic</a>
+                <a class="dropdown-item" href="#">Target</a>
+                <a class="dropdown-item" href="#">Lexington Station</a>
+                <a class="dropdown-item" href="#">Cub Foods</a>
```

Comment:
It would be better to get these values from the database and send them over to html using jinja variables so if the locations ever change we don't have to update the code.

Reply:
In my current project, I made locations_list.html to hold those values and then input them using {% block arriving_locations %}{% include 'locations_list.html' %}{% endblock %} to the different pages to keep from repeating. Is that ok or is database better? And if db is better, where would I grab them from?

Reply:
I guess you should probably just grab them from the spreadsheet.

Reply:
Ahh that's a good idea